.github/workflows/reusable_terraform_server.yml

```diff
@@ -105,6 +105,29 @@ jobs:
           path: ./infrastructure/server/fam-ui-api.zip
           if-no-files-found: error
 
+      - name: Install and Package Dependencies - FAM User Management API
```

Comment:
Just wondering, do we still call "User Management API", not admin management?

Reply:
forgot to change